webapp/src/css/inbox.less

```diff
@@ -1443,7 +1455,8 @@ mm-search-bar {
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
-      width: 100%;
+      width: 87%; // Give space to "more options" menu
```

Comment:
because this text has an ellipsis, it needs a fixed percentage for `width`. I tried using `calc(100vw - Xpx)` but then the ellipsis stops working.

Reply:
With which chrome version have you tried `calc(100vw - Xpx)`? It works with the latest version but I don't have an older version to try it out with

Reply:
It says:
```
Chrome is up to date
Version 108.0.5359.124 (Official Build) (x86_64)
```
:( 
I just try again but not luck... did you put something else?

Reply:
I don't have anything special installed in my chrome

Reply:
Maybe I'm looking at the wrong `.ellipsis-title` :sweat_smile:  
I made a quick recording of what I'm testing, let me know if that helps

https://user-images.githubusercontent.com/13026820/207842888-c806d8f2-2bf1-4ec8-be66-d09777042f00.mp4

Reply:
I need to consider the title's left + right spaces, not just the right side space 🤦🏽‍♀️
Fixed!

Reply:
btw I tried smaller values like 80px but doesn't fit nicely for small devices, so I kept the 100px.